### DISCUSSION TITLE
RUM nejede v prohlížečích jako je Arc

### DISCUSSION THREAD

**Discussion by lucienevludova-direct** [canarytrace/documentation#217] [RUM]
Ahoj, 

od kolegy předávám informaci z callu v minulém týdnu. Chyba způsobující, že RUM nejede v prohlížečích jako je Arc, ve skriptu rum.js, tj:
```
for (let ven of userAgentData.brands) {
        **if ("Chromium" != ven.brand && !ven.brand.includes("Not"))**
            return _nUA.brand = ven.brand,
            _nUA.version = ven.version,
            _nUA
        }
```
 
Ta zvýrazněná podmínka způsobí, že to právě nejede na tom Arcu a asi ani jiných prohlížečích, co jedou nad Chromium jádrem. Tam to totiž spadne na tom "Not", protože se testuje, aby ho neobsahoval žádný prvek v poli userAgentData.brands. Jenže ten je v tomhle případě
[{brand: 'Chromium', version: '127'},{brand: 'Not)A;Brand', version: '99'}]

**Comment by rdpanek** [comment]
Ahoj, 
díky za zprávu. Zaznamenali jste ještě nějaký jiný konkrétní a majoritně používaný prohlížeč ve kterém by RUM nepracovalo? Arc si píšu a mrkneme na něj, jak se přesně identifikuje.

Díky

**Reply by lucienevludova-direct** [reply]
Ahoj, zatím jsme narazili jen na Arc, s tím, že jsme zkoušeli RUM ještě v Chrome, Edge, Vivaldi, Brave a Safari.
